src/Form/LegacyConsumer/Commands/SetupFieldEmailTag.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Give\Form\LegacyConsumer\Commands;
+
+use Give\Framework\FieldsAPI\FormField;
+use Give\Framework\FieldsAPI\FieldCollection;
+use Give\Form\LegacyConsumer\FieldView;
+
+/**
+ * @unreleased
+ */
+class SetupFieldEmailTag {
```

Comment:
How can we test these classes? Some integration testing here would be nice. In this case we could use the `give_do_email_tags` method to test the resolver on some content. Without testing this we only know that the field email tag property is being set.

Reply:
Resolved by https://github.com/impress-org/givewp/pull/5649/commits/c84a9bdc0bdd18107d9fce8d2f7cd2104a3e55b4.